build/nsm/docker/Dockerfile

```diff
@@ -8,4 +8,5 @@ RUN ./scripts/build.sh --race-test-disabled
 FROM alpine as runtime
 COPY --from=build /go/bin/netmesh /go/bin/netmesh
 RUN mkdir -p /var/lib/kubelet/device-plugins
-ENTRYPOINT ["/go/bin/netmesh", "-microservice-label=netmesh", "-kube-config=/conf/kube.conf", "-http-config=/conf/http.conf"]
+ENV GOTRACEBACK=system
+ENTRYPOINT ["/go/bin/netmesh", "--kube-config=/conf/kube.conf"]
```

Comment:
I am not sure if it is right approach to provide arguments hard coded into the image. These arguments should be passed in a yaml file of the daemonset.

Reply:
@sbezverk are you referring to the GOTRACEBACK here or the --kube-config?

If --kube-config ... I'm just retaining existing behavior, would be happy to discuss changing it, but have tried to limit changes in this PR to what is strictly necessary to make the transition.